web/src/settings_display.js

```diff
@@ -103,11 +100,11 @@ function user_default_language_modal_post_render() {
                 setting_value,
             );
 
-            change_display_setting(
+            settings_ui.do_default_language_setting_change(
```

Comment:
There is no need for a separate function here. You can directly call channel.patch.

Also, please fix the commit history here and remove unnecessary commits. See https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html for details.

Reply:
Done here: 2d756845f5d2286b7be5fb8ee0cc36013d83603d.